server/resources/default-configs.properties

```diff
@@ -67,4 +67,5 @@ gremlin.spark.persistContext=true
 spark.master=local[*]
 spark.serializer=org.apache.spark.serializer.KryoSerializer
 spark.kryo.registrator=org.apache.tinkerpop.gremlin.spark.structure.io.gryo.GryoRegistrator
-spark.ui.showConsoleProgress=false
\ No newline at end of file
+spark.ui.showConsoleProgress=false
+spark.ui.enabled=false
```

Comment:
is this needed?

Reply:
yes apparently (discovered via VisualVM) the Spark UI creates a lot of threads (~10) when we use analytics queries, given that we never access the Spark web UI dashboard in production it's probably best to disable it for the time being